frontend/apps/os-shell/src/pages/workbench/__tests__/WorkbenchTabBar.test.tsx

```
@@ -0,0 +1,346 @@
+
+  describe('Tab Order Constitutional Invariants', () => {
+    it('tab_order_is_locked_and_constitutional', async () => {
+      renderWorkbench();
+
+      await waitFor(() => {
+        expect(screen.getByText('Summary')).toBeInTheDocument();
+      });
+
+      // Get all nav links
+      const tabLinks = screen.getAllByRole('link');
+      const tabLabels = tabLinks.map((link) => link.textContent?.replace(/[📊🔥🗺️📋📁🎮]/g, '').trim());
```

Biome (2.3.13)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)